Fixes for build --minimal

ptomato opened the pull request:
#### cfc4a23cc201b67e7023ffd34cdb1245ac4a60b3
<pre>
Fixes for build --minimal
<a href="https://bugs.webkit.org/show_bug.cgi?id=263905">https://bugs.webkit.org/show_bug.cgi?id=263905</a>

Reviewed by Michael Catanzaro.

Header file additions:
These additions all seem to be cases where normally the header is
included
by another header, but when compiling with a certain feature turned off,
the header no longer gets pulled in, making some function or class used
in
the file undeclared.

This especially sometimes shows up as linker errors, when a
&quot;FooInlines.h&quot;
header doesn&apos;t get pulled in, and an inline function is declared in the
regular &quot;Foo.h&quot; header but its body is never included.

These all show up as compiler or linker errors when building WebKit with
--minimal.

In some cases, remove a forward declaration since we do actually need to
include its header.

Preprocessor guards:

These changes fix builds with various options set by --minimal, such as
--no-video, --no-layer-based-svg-engine, --no-context-menus, etc.

In some cases, add UNUSED_PARAM to fix a warning that only occurs with
--no-video. In other cases, these changes fix compile errors.

Note that ENABLE_MODERN_MEDIA_CONTROLS is also switched off if
ENABLE_VIDEO is off.

* Source/WebCore/Modules/WebGPU/GPUExternalTextureDescriptor.h:
(WebCore::GPUExternalTextureDescriptor::convertToBacking const):
* Source/WebCore/Modules/WebGPU/InternalAPI/WebGPUExternalTextureDescriptor.h:
* Source/WebCore/accessibility/AccessibilityObject.cpp:
* Source/WebCore/bindings/js/JSWorkletGlobalScopeBase.h:
* Source/WebCore/dom/CDATASection.cpp:
* Source/WebCore/dom/ContentVisibilityDocumentState.cpp:
* Source/WebCore/dom/CustomElementRegistry.h:
* Source/WebCore/dom/Document.cpp:
(WebCore::Document::urlForBindings const):
* Source/WebCore/dom/DocumentFontLoader.cpp:
* Source/WebCore/dom/DocumentMarkerController.cpp:
* Source/WebCore/dom/IdTargetObserverRegistry.h:
* Source/WebCore/dom/NamedNodeMap.h:
* Source/WebCore/dom/PendingScript.h:
* Source/WebCore/dom/ScriptedAnimationController.h:
* Source/WebCore/editing/AlternativeTextController.cpp:
* Source/WebCore/editing/DictationCommand.cpp:
* Source/WebCore/editing/SpellingCorrectionCommand.h:
* Source/WebCore/editing/SplitTextNodeCommand.cpp:
* Source/WebCore/editing/VisibleSelection.cpp:
* Source/WebCore/editing/VisibleUnits.cpp:
* Source/WebCore/history/CachedPage.cpp:
* Source/WebCore/html/CheckboxInputType.cpp:
* Source/WebCore/html/HiddenInputType.cpp:
* Source/WebCore/html/SubmitInputType.cpp:
* Source/WebCore/html/parser/HTMLSrcsetParser.cpp:
* Source/WebCore/html/parser/HTMLSrcsetParser.h:
* Source/WebCore/html/shadow/SwitchThumbElement.cpp:
* Source/WebCore/html/shadow/SwitchTrackElement.cpp:
* Source/WebCore/layout/formattingContexts/inline/InlineLine.cpp:
(WebCore::Layout::Line::Run::isContentfulOrHasDecoration):
* Source/WebCore/layout/formattingContexts/inline/TextOnlySimpleLineBuilder.cpp:
(WebCore::Layout::TextOnlySimpleLineBuilder::isEligibleForSimplifiedTextOnlyInlineLayout):
* Source/WebCore/loader/archive/Archive.cpp:
* Source/WebCore/loader/cache/CachedResourceLoader.cpp:
(WebCore::CachedResourceLoader::updateRequestAfterRedirection):
* Source/WebCore/page/DOMWindow.cpp:
* Source/WebCore/page/LocalDOMWindow.cpp:
* Source/WebCore/page/LocalFrame.cpp:
* Source/WebCore/page/Navigation.cpp:
* Source/WebCore/page/NavigationDestination.cpp:
* Source/WebCore/page/NavigationHistoryEntry.cpp:
* Source/WebCore/platform/glib/UserAgentQuirks.cpp:
* Source/WebCore/platform/graphics/FontMetrics.h:
* Source/WebCore/rendering/LayoutRepainter.cpp:
* Source/WebCore/rendering/MarkedText.cpp:
* Source/WebCore/rendering/RenderFileUploadControl.cpp:
* Source/WebCore/rendering/RenderLayer.cpp:
(WebCore::RenderLayer::paintSVGResourceLayer):
(WebCore::RenderLayer::setupClipPath):
* Source/WebCore/rendering/RenderLayerModelObject.cpp:
* Source/WebCore/rendering/RenderReplaced.cpp:
(WebCore::isVideoWithDefaultObjectSize):
* Source/WebCore/rendering/RenderThemeAdwaita.cpp:
* Source/WebCore/rendering/TextBoxPainter.cpp:
* Source/WebCore/rendering/TransformOperationData.cpp:
(WebCore::TransformOperationData::TransformOperationData):
* Source/WebCore/rendering/style/WillChangeData.cpp:
(WebCore::WillChangeData::createsContainingBlockForOutOfFlowPositioned const):
* Source/WebCore/rendering/svg/RenderSVGResourceClipper.cpp:
* Source/WebCore/rendering/svg/RenderSVGResourceClipper.h:
* Source/WebCore/rendering/svg/RenderSVGResourceContainer.cpp:
* Source/WebCore/rendering/svg/RenderSVGResourceContainer.h:
* Source/WebCore/rendering/svg/SVGRenderSupport.cpp:
(WebCore::SVGRenderSupport::paintSVGClippingMask):
* Source/WebCore/rendering/svg/SVGResources.cpp:
(WebCore::SVGResources::buildCachedResources):
* Source/WebCore/svg/SVGClipPathElement.cpp:
* Source/WebCore/testing/InternalSettings.cpp:
(WebCore::InternalSettings::setShouldDeactivateAudioSession):
* Source/WebCore/testing/Internals.cpp:
(WebCore::Internals::isMonitoringWirelessRoutes const):
(WebCore::Internals::setIsPlayingToAutomotiveHeadUnit):
* Source/WebCore/workers/service/context/ServiceWorkerThreadProxy.cpp:
(WebCore::ServiceWorkerThreadProxy::setInspectable):
* Source/WebKit/NetworkProcess/NetworkProcess.h:
* Source/WebKit/NetworkProcess/NetworkSession.h:
* Source/WebKit/Platform/IPC/Encoder.h:
* Source/WebKit/Shared/API/APICaptionUserPreferencesTestingModeToken.h:
* Source/WebKit/Shared/API/glib/WebKitContextMenu.cpp:
* Source/WebKit/Shared/API/glib/WebKitContextMenuActions.cpp:
* Source/WebKit/Shared/API/glib/WebKitContextMenuActionsPrivate.h:
* Source/WebKit/Shared/API/glib/WebKitContextMenuItem.cpp:
(webkit_context_menu_item_new):
(webkit_context_menu_item_new_from_gaction):
(webkit_context_menu_item_new_from_stock_action):
(webkit_context_menu_item_new_from_stock_action_with_label):
(webkit_context_menu_item_new_with_submenu):
(webkit_context_menu_item_new_separator):
(webkit_context_menu_item_get_action):
(webkit_context_menu_item_get_gaction):
(webkit_context_menu_item_get_stock_action):
(webkit_context_menu_item_is_separator):
(webkit_context_menu_item_set_submenu):
(webkit_context_menu_item_get_submenu):
* Source/WebKit/Shared/API/glib/WebKitContextMenuItemPrivate.h:
* Source/WebKit/Shared/API/glib/WebKitContextMenuPrivate.h:
* Source/WebKit/Shared/glib/WebContextMenuItemGlib.cpp:
* Source/WebKit/Shared/glib/WebContextMenuItemGlib.h:
* Source/WebKit/UIProcess/API/glib/WebKitContextMenuClient.cpp:
* Source/WebKit/UIProcess/API/glib/WebKitContextMenuClient.h:
* Source/WebKit/UIProcess/API/glib/WebKitPointerLockPermissionRequest.cpp:
(webkit_permission_request_interface_init):
(webkitPointerLockPermissionRequestDispose):
* Source/WebKit/UIProcess/API/glib/WebKitWebView.cpp:
(webkitWebViewConstructed):
(webkit_web_view_save):
(webkit_web_view_save_to_file):
* Source/WebKit/UIProcess/API/gtk/PageClientImpl.cpp:
* Source/WebKit/UIProcess/API/gtk/PageClientImpl.h:
* Source/WebKit/UIProcess/API/gtk/WebKitColorChooser.cpp:
* Source/WebKit/UIProcess/API/gtk/WebKitColorChooser.h:
* Source/WebKit/UIProcess/API/gtk/WebKitColorChooserRequest.cpp:
(webkit_color_chooser_request_get_element_rectangle):
(webkit_color_chooser_request_cancel):
* Source/WebKit/UIProcess/API/gtk/WebKitColorChooserRequestPrivate.h:
* Source/WebKit/UIProcess/API/gtk/WebKitWebViewBase.cpp:
(webkitWebViewBaseHandleMouseEvent):
(webkit_web_view_base_class_init):
(webkitWebViewBaseSynthesizeMouseEvent):
(webkitWebViewBaseSynthesizeKeyEvent):
* Source/WebKit/UIProcess/API/gtk/WebKitWebViewBasePrivate.h:
* Source/WebKit/UIProcess/Network/NetworkProcessProxy.h:
* Source/WebKit/UIProcess/RemotePageProxy.h:
* Source/WebKit/UIProcess/WebProcessProxy.cpp:
* Source/WebKit/UIProcess/WebProcessProxy.h:
* Source/WebKit/UIProcess/WebsiteData/WebsiteDataStore.cpp:
(WebKit::WebsiteDataStore::updateServiceWorkerInspectability):
* Source/WebKit/UIProcess/WebsiteData/WebsiteDataStore.h:
* Source/WebKit/WebProcess/InjectedBundle/API/c/WKBundlePage.cpp:
(WKBundlePageCreateCaptionUserPreferencesTestingModeToken):
* Source/WebKit/WebProcess/InjectedBundle/API/glib/WebKitWebPage.cpp:
(webkit_web_page_class_init):
(webkitWebPageCreate):
* Source/WebKit/WebProcess/Storage/WebSWContextManagerConnection.cpp:
(WebKit::WebSWContextManagerConnection::installServiceWorker):
* Source/WebKit/WebProcess/WebCoreSupport/WebGeolocationClient.h:
* Source/WebKit/WebProcess/WebPage/CoordinatedGraphics/DrawingAreaCoordinatedGraphics.cpp:
(WebKit::DrawingAreaCoordinatedGraphics::registerScrollingTree):
(WebKit::DrawingAreaCoordinatedGraphics::unregisterScrollingTree):
* Source/WebKit/WebProcess/WebPage/EventDispatcher.h:

Canonical link: <a href="https://commits.webkit.org/270103@main">https://commits.webkit.org/270103@main</a>
</pre>
<!--EWS-Status-Bubble-Start-->
https://github.com/WebKit/WebKit/commit/8b1bc367bad52b542224c2613365f339bdd77b46

| Misc | iOS, tvOS & watchOS  | macOS  | Linux |  Windows |
| ----- | ---------------------- | ------- |  ----- |  --------- |
| [✅ 🧪 style](https://ews-build.webkit.org/#/builders/38/builds/24563 "Passed style check") | [✅ 🛠 ios](https://ews-build.webkit.org/#/builders/48/builds/3112 "Built successfully") | [✅ 🛠 mac](https://ews-build.webkit.org/#/builders/14/builds/25822 "Built successfully") | [✅ 🛠 wpe](https://ews-build.webkit.org/#/builders/5/builds/26688 "Built successfully") | [✅ 🛠 wincairo](https://ews-build.webkit.org/#/builders/32/builds/22574 "Built successfully") 
| [✅ 🧪 bindings](https://ews-build.webkit.org/#/builders/9/builds/24832 "Passed tests") | [✅ 🛠 ios-sim](https://ews-build.webkit.org/#/builders/49/builds/4759 "Built successfully") | [✅ 🛠 mac-AS-debug](https://ews-build.webkit.org/#/builders/51/builds/545 "Built successfully") | [  ~~🧪 wpe-wk2~~](https://ews-build.webkit.org/#/builders/34/builds/22964 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | 
| [✅ 🧪 webkitperl](https://ews-build.webkit.org/#/builders/11/builds/24808 "Passed tests") | [✅ 🧪 ios-wk2](https://ews-build.webkit.org/#/builders/47/builds/2165 "Passed tests") | [✅ 🧪 api-mac](https://ews-build.webkit.org/#/builders/18/builds/21218 "Passed tests") | [✅ 🛠 gtk](https://ews-build.webkit.org/#/builders/2/builds/27271 "Built successfully") | 
| | [✅ 🧪 ios-wk2-wpt](https://ews-build.webkit.org/#/builders/42/builds/1896 "Passed tests") | [✅ 🧪 mac-wk1](https://ews-build.webkit.org/#/builders/10/builds/22144 "Passed tests") | [✅ 🧪 gtk-wk2](https://ews-build.webkit.org/#/builders/1/builds/28330 "Passed tests") | 
| | [✅ 🧪 api-ios](https://ews-build.webkit.org/#/builders/13/builds/22393 "Passed tests") | [✅ 🧪 mac-wk2](https://ews-build.webkit.org/#/builders/36/builds/22492 "Passed tests") | [✅ 🧪 api-gtk](https://ews-build.webkit.org/#/builders/21/builds/26123 "Passed tests") | 
| | [✅ 🛠 tv](https://ews-build.webkit.org/#/builders/44/builds/1827 "Built successfully") | [✅ 🧪 mac-AS-debug-wk2](https://ews-build.webkit.org/#/builders/50/builds/153 "Passed tests") | | 
| | [✅ 🛠 tv-sim](https://ews-build.webkit.org/#/builders/46/builds/3138 "Built successfully") | | | 
| [✅ 🛠 🧪 merge](https://ews-build.webkit.org/#/builders/19/builds/5886 "Built successfully and passed tests") | [✅ 🛠 watch](https://ews-build.webkit.org/#/builders/43/builds/2284 "Built successfully") | | | 
| | [✅ 🛠 watch-sim](https://ews-build.webkit.org/#/builders/45/builds/2200 "Built successfully") | | | 
<!--EWS-Status-Bubble-End-->

mcatanzaro commented on Source/WebCore/rendering/RenderReplaced.cpp:
```diff
@@ -699,9 +702,10 @@ void RenderReplaced::computePreferredLogicalWidths()
+    // We cannot resolve some logical width here (i.e. percent, fill-available or fit-content)
+    // as the available logical width may not be set on our containing block.
+    const auto& logicalWidth = style().logicalWidth();
+    if (logicalWidth.isPercentOrCalculated() || logicalWidth.isFillAvailable() || logicalWidth.isFitContent())
```
This looks like a mistake, right? Surely this is intended for another pull request?

ptomato replied:
Sorry about that! I have no idea how this got in. Must have been a mistake while I was rebasing. I'll scroll through it one more time and make sure there's nothing else like this.

mcatanzaro commented on Source/WebKit/WebProcess/InjectedBundle/API/glib/WebKitWebPage.cpp:
```diff
@@ -601,7 +603,11 @@ static void webkit_web_page_class_init(WebKitWebPageClass* klass)
         g_signal_accumulator_true_handled, nullptr,
         g_cclosure_marshal_generic,
         G_TYPE_BOOLEAN, 2,
+#if ENABLE(CONTEXT_MENUS)
         WEBKIT_TYPE_CONTEXT_MENU,
+#else
+        G_TYPE_OBJECT,
+#endif // ENABLE(CONTEXT_MENUS)
```
I'm not sure if the introspection scanner can handle this or not. Let's give it a try.

ptomato replied:
An alternative would be to just disallow `--no-context-menus` with the GTK port. Let me know if I should pursue that. (But if this `#else` doesn't cause problems, I'd prefer to do that in a separate PR)

mcatanzaro replied:
Well, the EWS are happy, so it's probably fine.

ptomato committed:
Fixes for build --minimal
https://bugs.webkit.org/show_bug.cgi?id=263905

Reviewed by Michael Catanzaro.

Header file additions:
These additions all seem to be cases where normally the header is
included
by another header, but when compiling with a certain feature turned off,
the header no longer gets pulled in, making some function or class used
in
the file undeclared.

This especially sometimes shows up as linker errors, when a
"FooInlines.h"
header doesn't get pulled in, and an inline function is declared in the
regular "Foo.h" header but its body is never included.

These all show up as compiler or linker errors when building WebKit with
--minimal.

In some cases, remove a forward declaration since we do actually need to
include its header.

Preprocessor guards:

These changes fix builds with various options set by --minimal, such as
--no-video, --no-layer-based-svg-engine, --no-context-menus, etc.

In some cases, add UNUSED_PARAM to fix a warning that only occurs with
--no-video. In other cases, these changes fix compile errors.

Note that ENABLE_MODERN_MEDIA_CONTROLS is also switched off if
ENABLE_VIDEO is off.

* Source/WebCore/Modules/WebGPU/GPUExternalTextureDescriptor.h:
(WebCore::GPUExternalTextureDescriptor::convertToBacking const):
* Source/WebCore/Modules/WebGPU/InternalAPI/WebGPUExternalTextureDescriptor.h:
* Source/WebCore/accessibility/AccessibilityObject.cpp:
* Source/WebCore/bindings/js/JSWorkletGlobalScopeBase.h:
* Source/WebCore/dom/CDATASection.cpp:
* Source/WebCore/dom/ContentVisibilityDocumentState.cpp:
* Source/WebCore/dom/CustomElementRegistry.h:
* Source/WebCore/dom/Document.cpp:
(WebCore::Document::urlForBindings const):
* Source/WebCore/dom/DocumentFontLoader.cpp:
* Source/WebCore/dom/DocumentMarkerController.cpp:
* Source/WebCore/dom/IdTargetObserverRegistry.h:
* Source/WebCore/dom/NamedNodeMap.h:
* Source/WebCore/dom/PendingScript.h:
* Source/WebCore/dom/ScriptedAnimationController.h:
* Source/WebCore/editing/AlternativeTextController.cpp:
* Source/WebCore/editing/DictationCommand.cpp:
* Source/WebCore/editing/SpellingCorrectionCommand.h:
* Source/WebCore/editing/SplitTextNodeCommand.cpp:
* Source/WebCore/editing/VisibleSelection.cpp:
* Source/WebCore/editing/VisibleUnits.cpp:
* Source/WebCore/history/CachedPage.cpp:
* Source/WebCore/html/CheckboxInputType.cpp:
* Source/WebCore/html/HiddenInputType.cpp:
* Source/WebCore/html/SubmitInputType.cpp:
* Source/WebCore/html/parser/HTMLSrcsetParser.cpp:
* Source/WebCore/html/parser/HTMLSrcsetParser.h:
* Source/WebCore/html/shadow/SwitchThumbElement.cpp:
* Source/WebCore/html/shadow/SwitchTrackElement.cpp:
* Source/WebCore/layout/formattingContexts/inline/InlineLine.cpp:
(WebCore::Layout::Line::Run::isContentfulOrHasDecoration):
* Source/WebCore/layout/formattingContexts/inline/TextOnlySimpleLineBuilder.cpp:
(WebCore::Layout::TextOnlySimpleLineBuilder::isEligibleForSimplifiedTextOnlyInlineLayout):
* Source/WebCore/loader/archive/Archive.cpp:
* Source/WebCore/loader/cache/CachedResourceLoader.cpp:
(WebCore::CachedResourceLoader::updateRequestAfterRedirection):
* Source/WebCore/page/DOMWindow.cpp:
* Source/WebCore/page/LocalDOMWindow.cpp:
* Source/WebCore/page/LocalFrame.cpp:
* Source/WebCore/page/Navigation.cpp:
* Source/WebCore/page/NavigationDestination.cpp:
* Source/WebCore/page/NavigationHistoryEntry.cpp:
* Source/WebCore/platform/glib/UserAgentQuirks.cpp:
* Source/WebCore/platform/graphics/FontMetrics.h:
* Source/WebCore/rendering/LayoutRepainter.cpp:
* Source/WebCore/rendering/MarkedText.cpp:
* Source/WebCore/rendering/RenderFileUploadControl.cpp:
* Source/WebCore/rendering/RenderLayer.cpp:
(WebCore::RenderLayer::paintSVGResourceLayer):
(WebCore::RenderLayer::setupClipPath):
* Source/WebCore/rendering/RenderLayerModelObject.cpp:
* Source/WebCore/rendering/RenderReplaced.cpp:
(WebCore::isVideoWithDefaultObjectSize):
* Source/WebCore/rendering/RenderThemeAdwaita.cpp:
* Source/WebCore/rendering/TextBoxPainter.cpp:
* Source/WebCore/rendering/TransformOperationData.cpp:
(WebCore::TransformOperationData::TransformOperationData):
* Source/WebCore/rendering/style/WillChangeData.cpp:
(WebCore::WillChangeData::createsContainingBlockForOutOfFlowPositioned const):
* Source/WebCore/rendering/svg/RenderSVGResourceClipper.cpp:
* Source/WebCore/rendering/svg/RenderSVGResourceClipper.h:
* Source/WebCore/rendering/svg/RenderSVGResourceContainer.cpp:
* Source/WebCore/rendering/svg/RenderSVGResourceContainer.h:
* Source/WebCore/rendering/svg/SVGRenderSupport.cpp:
(WebCore::SVGRenderSupport::paintSVGClippingMask):
* Source/WebCore/rendering/svg/SVGResources.cpp:
(WebCore::SVGResources::buildCachedResources):
* Source/WebCore/svg/SVGClipPathElement.cpp:
* Source/WebCore/testing/InternalSettings.cpp:
(WebCore::InternalSettings::setShouldDeactivateAudioSession):
* Source/WebCore/testing/Internals.cpp:
(WebCore::Internals::isMonitoringWirelessRoutes const):
(WebCore::Internals::setIsPlayingToAutomotiveHeadUnit):
* Source/WebCore/workers/service/context/ServiceWorkerThreadProxy.cpp:
(WebCore::ServiceWorkerThreadProxy::setInspectable):
* Source/WebKit/NetworkProcess/NetworkProcess.h:
* Source/WebKit/NetworkProcess/NetworkSession.h:
* Source/WebKit/Platform/IPC/Encoder.h:
* Source/WebKit/Shared/API/APICaptionUserPreferencesTestingModeToken.h:
* Source/WebKit/Shared/API/glib/WebKitContextMenu.cpp:
* Source/WebKit/Shared/API/glib/WebKitContextMenuActions.cpp:
* Source/WebKit/Shared/API/glib/WebKitContextMenuActionsPrivate.h:
* Source/WebKit/Shared/API/glib/WebKitContextMenuItem.cpp:
(webkit_context_menu_item_new):
(webkit_context_menu_item_new_from_gaction):
(webkit_context_menu_item_new_from_stock_action):
(webkit_context_menu_item_new_from_stock_action_with_label):
(webkit_context_menu_item_new_with_submenu):
(webkit_context_menu_item_new_separator):
(webkit_context_menu_item_get_action):
(webkit_context_menu_item_get_gaction):
(webkit_context_menu_item_get_stock_action):
(webkit_context_menu_item_is_separator):
(webkit_context_menu_item_set_submenu):
(webkit_context_menu_item_get_submenu):
* Source/WebKit/Shared/API/glib/WebKitContextMenuItemPrivate.h:
* Source/WebKit/Shared/API/glib/WebKitContextMenuPrivate.h:
* Source/WebKit/Shared/glib/WebContextMenuItemGlib.cpp:
* Source/WebKit/Shared/glib/WebContextMenuItemGlib.h:
* Source/WebKit/UIProcess/API/glib/WebKitContextMenuClient.cpp:
* Source/WebKit/UIProcess/API/glib/WebKitContextMenuClient.h:
* Source/WebKit/UIProcess/API/glib/WebKitPointerLockPermissionRequest.cpp:
(webkit_permission_request_interface_init):
(webkitPointerLockPermissionRequestDispose):
* Source/WebKit/UIProcess/API/glib/WebKitWebView.cpp:
(webkitWebViewConstructed):
(webkit_web_view_save):
(webkit_web_view_save_to_file):
* Source/WebKit/UIProcess/API/gtk/PageClientImpl.cpp:
* Source/WebKit/UIProcess/API/gtk/PageClientImpl.h:
* Source/WebKit/UIProcess/API/gtk/WebKitColorChooser.cpp:
* Source/WebKit/UIProcess/API/gtk/WebKitColorChooser.h:
* Source/WebKit/UIProcess/API/gtk/WebKitColorChooserRequest.cpp:
(webkit_color_chooser_request_get_element_rectangle):
(webkit_color_chooser_request_cancel):
* Source/WebKit/UIProcess/API/gtk/WebKitColorChooserRequestPrivate.h:
* Source/WebKit/UIProcess/API/gtk/WebKitWebViewBase.cpp:
(webkitWebViewBaseHandleMouseEvent):
(webkit_web_view_base_class_init):
(webkitWebViewBaseSynthesizeMouseEvent):
(webkitWebViewBaseSynthesizeKeyEvent):
* Source/WebKit/UIProcess/API/gtk/WebKitWebViewBasePrivate.h:
* Source/WebKit/UIProcess/Network/NetworkProcessProxy.h:
* Source/WebKit/UIProcess/RemotePageProxy.h:
* Source/WebKit/UIProcess/WebProcessProxy.cpp:
* Source/WebKit/UIProcess/WebProcessProxy.h:
* Source/WebKit/UIProcess/WebsiteData/WebsiteDataStore.cpp:
(WebKit::WebsiteDataStore::updateServiceWorkerInspectability):
* Source/WebKit/UIProcess/WebsiteData/WebsiteDataStore.h:
* Source/WebKit/WebProcess/InjectedBundle/API/c/WKBundlePage.cpp:
(WKBundlePageCreateCaptionUserPreferencesTestingModeToken):
* Source/WebKit/WebProcess/InjectedBundle/API/glib/WebKitWebPage.cpp:
(webkit_web_page_class_init):
(webkitWebPageCreate):
* Source/WebKit/WebProcess/Storage/WebSWContextManagerConnection.cpp:
(WebKit::WebSWContextManagerConnection::installServiceWorker):
* Source/WebKit/WebProcess/WebCoreSupport/WebGeolocationClient.h:
* Source/WebKit/WebProcess/WebPage/CoordinatedGraphics/DrawingAreaCoordinatedGraphics.cpp:
(WebKit::DrawingAreaCoordinatedGraphics::registerScrollingTree):
(WebKit::DrawingAreaCoordinatedGraphics::unregisterScrollingTree):
* Source/WebKit/WebProcess/WebPage/EventDispatcher.h:

Canonical link: https://commits.webkit.org/270103@main
